domain-model-assistant/Assets/Components/Scripts/lineCollider.cs

```diff
@@ -25,45 +31,43 @@ void Start()
     {
         edge = GetComponent<Edge>();
         polygonCollider2D = GetComponent<PolygonCollider2D>();
-        highlightBox = new GameObject().AddComponent<LineRenderer>() as LineRenderer;
-        highlightBox.SetColors(c1,c1);
-        highlightBox.enabled = false;
-    
     }
 
-    private void OnDrawGizmos() 
-    {
-        Gizmos.color = Color.black;
-        if (colliderPoints != null) colliderPoints.ForEach(p => Gizmos.DrawSphere(p, 0.1f));
-    }
+
 
     // Update is called once per frame
     void Update()
     {
         colliderPoints = CalculateColliderPoints();
-    
         polygonCollider2D.SetPath(0, colliderPoints.ConvertAll(p => (Vector2)transform.InverseTransformPoint(p)));
-
     }
 
     
     //mouse hover, not clicked, set color to blue
     void OnMouseEnter()
     {  
-        Debug.Log("mouse enter");
-        edge.setColor(1);
+        if(!isSelected){
+            edge.setColor(1);
```

Comment:
Use formatting consistent with the project and C# guidelines. This applies to many locations, not just here. You can use the autoformatter to do this work for you

```cs
if (!isSelected) // space after if
{ // curly brace on new line
    edge.setColor(1);
```